### PR TITLE
readme: add prerequesite and logging, improve config

### PR DESCRIPTION
**Pull request by Mirko-von-Leipzig**
I'm not certain if this would be enough information for a novice -- but it might be enough for someone familiar with Zebra / ZCashd?

I've also removed the config field for setting the config file in anticipation of #85 getting merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +92,20 @@ Ziggurat's documentation can be built with `cargo doc --no-deps --open`.
 
 Ziggurat currently uses rust's standard test runner, a simple `cargo test -- --test-threads=1` should suffice. We use the single threaded executor as spinning up multiple test nodes isn't currently supported.
 
+### Logging
+
+By default, the node's `stdout` and `stderr` output will not be visible. One can enable this (in the code) on a per test basis by inserting a call to `log_to_stdout(true)` before starting the node. This can be used to see the node's logs on screen if appropriately configured.
```
Comment (niklaslong) [inline README.md:97]: Would be great to mention `enable_tracing` for synthetic nodes too?

**Review by niklaslong — COMMENTED** [review]
Nice! I think this is detailed enough, especially as most users of the project will be devs familiar with either zebrad, zcashd or both.

**Comment by niklaslong** [comment]
Clippy breakage may be due to https://github.com/rust-lang/rust-clippy/issues/7422.

File: README.md (hunk 1 above)
Comment (Mirko-von-Leipzig) [reply README.md:97]: > 
> 
> Clippy breakage may be due to [rust-lang/rust-clippy#7422](https://github.com/rust-lang/rust-clippy/issues/7422).

I think it is. It broke for me locally only after `rustup update`.

**Comment by Mirko-von-Leipzig** [comment]
> 
> 
> Clippy breakage may be due to [rust-lang/rust-clippy#7422](https://github.com/rust-lang/rust-clippy/issues/7422).

Yeah I think that's the cause. Only broke for me locally after `rustup update`

File: README.md (hunk 1 above)
Comment (Mirko-von-Leipzig) [reply README.md:97]: Thanks, I forgot about that!

**Review by niklaslong — APPROVED** [review]

**Comment by niklaslong** [comment]
Let's wait for #85 so we're in sync

**Pull request merged**